core/server/worker/src/main/java/alluxio/worker/netty/DataServerHandler.java

```diff
@@ -100,7 +100,8 @@ public void channelRead0(final ChannelHandlerContext ctx, final RPCMessage msg)
       }
     } catch (IllegalArgumentException | IOException e) {
```

Comment:
if we converted the body of the method into a callable we could use a the same logging utility here and in the other places we log